kroma-validator/challenger.go

```diff
@@ -641,30 +682,40 @@ func (c *Challenger) CanCreateChallenge(ctx context.Context, outputIndex *big.In
 			c.log.Warn("validator is not in the status that can create a challenge", "status", validatorStatus)
 			return false, nil
 		}
+
+		balance, err = c.assetMgrContract.TotalValidatorBalance(optsutils.NewSimpleCallOpts(cCtx), from)
+		if err != nil {
+			return false, fmt.Errorf("failed to fetch balance: %w", err)
+		}
+		requiredBondAmount = c.requiredBondAmountV2
 	} else {
-		balance, err := c.valPoolContract.BalanceOf(optsutils.NewSimpleCallOpts(cCtx), from)
+		var err error
+		balance, err = c.valPoolContract.BalanceOf(optsutils.NewSimpleCallOpts(cCtx), from)
 		if err != nil {
 			return false, fmt.Errorf("failed to fetch deposit amount: %w", err)
 		}
-		c.metr.RecordDepositAmount(balance)
-
-		if balance.Cmp(c.requiredBondAmount) == -1 {
-			c.log.Warn(
-				"deposit is less than bond attempt amount",
-				"requiredBondAmount", c.requiredBondAmount,
-				"deposit", balance,
-			)
-			return false, nil
-		}
+		requiredBondAmount = c.requiredBondAmountV1
+	}
 
-		c.log.Info("deposit amount and bond amount", "deposit", balance, "bond", c.requiredBondAmount)
+	c.metr.RecordDepositAmount(balance)
+
+	// Check if the deposit amount is less than the required bond amount
```

Comment:
Same in l2_output_submitter.
```suggestion
	// Check if the unbonded deposit amount is less than the required bond amount
```